<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-T6DNDPP" failures="1" tests="8" name="PetStoreTest" time="9.876" errors="0" timestamp="2023-07-17T20:13:54 PDT">
  <testcase classname="api.test.UserTest" name="testPostuser" time="3.048"/>
  <testcase classname="api.test.UserTest" name="TestUpdateUser" time="0.5"/>
  <testcase classname="api.test.UserTest" name="TestDeleteUser" time="0.474">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTest.TestDeleteUser(UserTest.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- TestDeleteUser -->
  <testcase classname="api.test.DDrTest" name="testpostUser" time="0.439"/>
  <testcase classname="api.test.DDrTest" name="testpostUser" time="0.437"/>
  <testcase classname="api.test.DDrTest" name="testpostUser" time="0.597"/>
  <testcase classname="api.test.DDrTest" name="testpostUser" time="0.662"/>
  <testcase classname="api.test.DDrTest" name="testpostUser" time="0.548"/>
</testsuite> <!-- PetStoreTest -->
